test/dotnet-project-convert.Tests/DotnetProjectConvertTests.cs

```diff
@@ -301,12 +301,12 @@ public void ProcessingSucceeds()
             .Should().Be("Console.WriteLine();");
 
         File.ReadAllText(Path.Join(testInstance.Path, "Program", "Program.csproj"))
-            .Should().Be("""
+            .Should().Be($"""
                 <Project Sdk="Aspire.Hosting.Sdk/9.1.0">
 
                   <PropertyGroup>
                     <OutputType>Exe</OutputType>
-                    <TargetFramework>net10.0</TargetFramework>
+                    <TargetFramework>{ToolsetInfo.CurrentTargetFramework}</TargetFramework>
```

Comment:
I am curious what happens when user writes `#:TargetFramework net9.0` or `#:TargetFramework net472` or what have you. Or, heck, `#:OutputType Library`. No need to test it here and now but perhaps eventually to demonstrate the behavior.

Reply:
There is a test for this - it's named `Directives`. The user values will be added to a PropertyGroup below, so MSBuild will deal with them and overwrite the built-ins, see also https://github.com/dotnet/sdk/pull/47702#discussion_r2003165286.